lib/File/pushd.pm

```diff
@@ -255,6 +255,75 @@ void context and the warnings category C<void> is enabled.
     pushd();
   }
 
+=head1 COMMOM MISTAKES
+
+=head2 Reusing a variable
+
+You should not reuse the same variable.
+
+When the PV is destroyed it's going to switch back to the original location.
+In this example the destruction occurs after the creation of the new object,
+and you would not be where you would expect to be.
+
+ {
+     # DO NOT DO THIS
+     chdir('/start');
+     my $dir = pushd( '/some/where' );
+     ...
+     # then later
+     $dir  = pushd( '/another/directory' );
+     # ..
+     # here you are back to '/start' directory
+ }
+
+You should use multiple scopes
+
+ {
+     # CORRECT WAY TO DO IT
+     chdir('/start');
+     my $dir = pushd( '/some/where' );
+     ...
+     {
+         # then later
+         my $dir  = pushd( '/another/directory' );
+         ...
+     }
+     # ..
+     # here you are back to '/start' directory
+ }
+
+=head2 Do not use twice pushd in the same scope
```

Comment:
Because this in a section called "COMMON MISTAKES", I think either both subheading should be phrased "Do not ..." or else neither should be.

Reply:
resubmitted with suggested change, I opt to do not use negation for head2 title